app/dao/notifications_dao.py

```diff
@@ -49,20 +55,22 @@ def dao_create_notification(notification, notification_type):
         raise
 
 
-def update_notification_stats(notification, notification_type):
-    if notification_type == TEMPLATE_TYPE_SMS:
-        update = {
-            NotificationStatistics.sms_requested: NotificationStatistics.sms_requested + 1
-        }
-    else:
-        update = {
-            NotificationStatistics.emails_requested: NotificationStatistics.emails_requested + 1
+def update_query(notification_type, status):
```

Comment:
Update queries mapped by type/outcome. Methods use notification type/status to pick correct update